src/main/fulcro/client/mutations.cljc

```diff
@@ -174,6 +174,26 @@
   (let [value (ensure-integer (if event (target-value event) value))]
     (set-value! component field value)))
 
+#?(:cljs
+   (defn- ensure-double
+     "Helper for set-integer!, use that instead. It is recommended you use this function only on UI-related
+     data (e.g. data that is used for display purposes) and write clear top-level transactions for anything else."
+     [v]
+     (let [rv (js/parseFloat v)]
+       (if (js/isNaN rv) 0 rv)))
+   :clj
+   (defn- ensure-double [v] (Double/parseDouble v)))
```

Comment:
This side will likely never get call, so this is nit-picky, but the JVM throws on bad number strings.

Reply:
Are you suggesting that we should catch and handle a bad value?
`set-integer!` does also throw on invalid strings.

Reply:
nah, it's fine.  The clj side should never be running anyway.  Isomorphic is for first-frame rendering in a headless env...so no way for it to ever run.  Even writing a body is overkill other than making tests easier to write.